source/manual/sentry.html.md

```diff
@@ -7,12 +7,215 @@ section: Monitoring
 type: learn
 ---
 
-We use [Sentry][] to notify us of exceptions that happen in all environments.
+[Sentry][] is a service that collates all of the errors that happen on GOV.UK
+in one place, regardless of which application, what environment or what
+machine the error occurred on. It is far more convenient than logging on to
+individual machines and querying their logs.
 
-We configure Sentry using [govuk-saas-config][]. It reads a list of apps from the docs and makes sure that all configuration is set up correctly.
+Useful links:
 
-Use the [govuk_app_config][] gem to configure your application to talk to Sentry.
+- [Sentry Organization Stats](https://sentry.io/organizations/govuk/stats/)
+  for an overview of how many errors are being sent/rate-limited site-wide.
+- [Sentry Projects](https://sentry.io/organizations/govuk/projects/)
+  for a per-project view, enabling you to see which ones are causing the
+  most errors right now.
+- Sentry Grafana dashboards:
+  [Production](https://grafana.production.govuk.digital/dashboard/file/sentry_errors_across_all_environments.json),
+  [Staging](https://grafana.staging.govuk.digital/dashboard/file/sentry_errors_across_all_environments.json)
+  and [Integration](https://grafana.integration.publishing.service.gov.uk/dashboard/file/sentry_errors_across_all_environments.json)
+  (all require VPN), for a per-environment breakdown of Sentry errors.
 
-[Sentry]: https://sentry.io/govuk
-[govuk-saas-config]: https://github.com/alphagov/govuk-saas-config/blob/5171b2803a7e211fff9536909b7d27c7fa5a4840/sentry/Rakefile
+[Sentry]: https://sentry.io/govuk/
+
+## Nomenclature
+
+- **Project**: each GOV.UK application has its own project on Sentry, where
+  errors originating from that application are consolidated. Example:
+  [publishing-api](https://sentry.io/organizations/govuk/issues/?project=202242)
+- **Issue**: a group of similar errors. For example, multiple instances of
+  `NoMethodError` from the same app are likely to be logged under the same
+  issue (see [example][example-issue]) in the project in Sentry. In practice,
+  there are a [variety of factors][anchor-error-grouping] that determine
+  whether errors are grouped under the same issue.
+- **Error/Exception**: a single occurrence of an Issue, containing details
+  such as stack trace, arguments, server IP and error message.
+  See [example][example-error].
+
+[anchor-error-grouping]: #fingerprinting
+[example-issue]: https://sentry.io/organizations/govuk/issues/2150349125/?project=202259&query=is%3Aunresolved
+[example-error]: https://sentry.io/organizations/govuk/issues/2155210524/events/aa4edd05a94c4d14b07d4c754a13c788/
+
+## How Sentry is integrated on GOV.UK
+
+Projects can be created and edited in the Sentry UI, but this risks creating
+inconsistencies or missing apps. We therefore configure projects using
+[govuk-saas-config][] (and its [associated rake tasks][rake-tasks]), which
```

Comment:
That's true, and is something I was encouraging everyone to do this time last year! Then we found that [the shortcut syntax didn't work with the Redcarpet markdown library](https://trello.com/c/XSgGyudd/181-fix-url-markdown-parsing) we were using and I stopped suggesting it. The issue is now fixed (by [moving to Kramdown](https://github.com/alphagov/govuk-developer-docs/pull/2785)) but I've since grown to prefer the explicit `[]` style.

Sometimes, shortcut links are followed by parentheses - `[foo] (https://google.com)` - which is confusing to look at. Is this a broken markdown link to google.com, where we need to remove the space between the brackets, or is it a working markdown link to an aliased `[foo]: http://something.else` later on in the document? It's hard to say without scanning the whole document, so I prefer the `[]` for the disambiguation. It's not something we've specified a preference for in the GDS Way, so it's up to us to muddle our way through as best we can!

Reply:
Ah, thanks for explaining on why you use the extra brackets, hadn't thought of the issue you mentioned - it makes sense!

> muddle our way through as best we can!

That's a common approach I use for general development! 😆